src/components/Header/Header.css

```diff
@@ -0,0 +1,30 @@
+header {
+  background-color: rgba(245, 245, 245, 0.90);
+  color: #2c3e50;
+  border-radius: 1rem;
+  padding: 2rem;
+}
+
+h1 {
+  margin: .5rem;
+}
+
+h2 {
+  font-size: x-large;
+}
+
+@media screen and (max-width: 600px) {
+  header {
+    width: 250px;
```

Comment:
Would we be able to use a percentage width instead of having two different set widths? I feel like it would be easier to handle varying screen sizes.

Reply:
Played with this, but leaving the options with set pixels as this allows the header to consistently line up better with the form component and cards below it.